src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -1333,7 +1333,7 @@ def create_lvs(volume_group, parts=None, size=None, name_prefix='ceph-lv'):
         size = sizing['sizes']
         extents = sizing['extents']
         lvs.append(
-            create_lv(name_prefix, uuid.uuid4(), vg=volume_group.name, extents=extents, tags=tags)
```

Comment:
Looks like a trivial type error. As you don't want to add Py2 type hints, when do you expect c-v will be py3 on master?

Reply:
whenever the last py2 release (nautilus iiuc) goes out of support. So after the P release, or Q?